### PR TITLE
Databases-Week-4_muhammet-emin-alper

### PR DESCRIPTION
**Pull request by eminalpernl**

### CODE REVIEW

**Comment by ArcoMul** [comment]
Hi @eminalpernl, please remove the homework files of the other weeks out of this Pull Request

**Comment by eminalpernl** [comment]
Hi Arco I went through all the Homeworks in this module.

File: Week4/homework/ex1-aggregation/index.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+
+import { MongoClient } from 'mongodb';
+import readCSV from 'csvtojson';
+import * as dotenv from 'dotenv' 
+dotenv.config()
+const URI = process.env.MONGODB_URL;
+
+async function main() {
+
+
```
Comment (ArcoMul) [inline Week4/homework/ex1-aggregation/index.js:10]: Please format your code using Prettier, double linebreaks shouldn't be in the file

File: Week4/homework/ex2-transactions/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  
+        const fromAccountChanges = fromAccount.account_changes;
+        const fromAccountChangeNumber = fromAccountChanges.length + 1;
+        fromAccountChanges.push({ change_number: fromAccountChangeNumber, amount: -amount, changed_date: timestamp, remark: remark });
+  
+        const toAccountChanges = toAccount.account_changes;
+        const toAccountChangeNumber = toAccountChanges.length + 1;
+        toAccountChanges.push({ change_number: toAccountChangeNumber, amount: amount, changed_date: timestamp, remark: remark });
+  
+        await accountsCollection.updateOne({ account_number: sender }, { $set: { balance: fromAccount.balance - amount, account_changes: fromAccountChanges } }, { session });
```
Comment (ArcoMul) [inline Week4/homework/ex2-transactions/index.js:50]: Please look into the `$inc` and `$push` parameters for `updateOne`. The code of this transaction can be shortened a lot using those features.

**Review by ArcoMul — CHANGES_REQUESTED** [review]
Hi @eminalpernl, thank you for the updates! Assignment one looks good, besides the formatting of the file. Exercise 2 can be improved a bit. Also please in this case, remove all the files which shouldn't be in this pull request (the ones from the other homework weeks). It makes it easier for me to review, and is a neater way of working in general.

**Pull request closed without merging**